apps/bynder-content-workflow/tsconfig.json

```diff
@@ -17,8 +17,8 @@
     "jsx": "react-jsx",
     "baseUrl": ".",
     "paths": {
-      "@/*": ["src/*"],
-    },
+      "@/*": ["src/*"]
```

Comment:
Would you mind removing this (I'm assuming accidental) changes to the bynder content workflow app?

Reply:
whoops done.